README.md

```diff
@@ -322,7 +322,7 @@ The Hiera naming substitutes underscore for any secondary double colons, i.e. sp
 Notes
 -----
 
-* Runs under Puppet 2.7 and later.
+* Runs under Puppet 4.9 and later.
```

Comment:
I expect this module to work on >=4.x but I am not sure what is the best method to validate this belief.